indico/htdocs/js/indico/modules/timetable/timetable/Draw.js

```diff
@@ -551,46 +548,123 @@ type("TimetableBlockWholeDayBase", ["TimetableBlockBase"],
      }
    );
 
+function drawBalloon (self, evt, editable) {
+    var timetableBlock = self.div.dom;
+    var entryId = self.eventData.scheduleEntryId ? self.eventData.scheduleEntryId : self.eventData.id.substring(1);
+    var params = {
+       'confId': self.eventData.conferenceId[0],
+       'timetable_entry_id': entryId
+    };
+    if(editable) {
+        params['editable'] = true;
+    }
+    var balloonId = '#qtip-' + entryId.toString();
+
+    $(timetableBlock).qbubble({
+       id: entryId.toString(),
+       content: {
+           text: function(event, api) {
+               $.ajax({
+                   url: build_url(Indico.Urls.Timetable.entries.balloon, params)
+               })
+               .then(function(content) {
```

Comment:
i'd put that in the previous line but i guess that's a matter of personal preference